src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -2033,6 +2033,18 @@ public static void Sort<T>(this Span<T> span, Comparison<T> comparison)
         /// <returns>The length of the common prefix shared by the two spans.  If there's no shared prefix, 0 is returned.</returns>
         public static int CommonPrefixLength<T>(this ReadOnlySpan<T> span, ReadOnlySpan<T> other)
         {
+            if (RuntimeHelpers.IsBitwiseEquatable<T>())
+            {
+                nuint length = Math.Min((nuint)(uint)span.Length, (nuint)(uint)other.Length);
```

Comment:
I was going to suggest `nuint.Min` but then I remembered that `IntPtr` and `UIntPtr` still have them explicitly implemented until the work around https://github.com/dotnet/csharplang/pull/6031 goes in.